packages/x-charts/src/hooks/useInteractionItemProps.ts

```diff
@@ -11,7 +11,12 @@ export const useInteractionItemProps = (skip?: boolean) => {
     return () => ({});
   }
   const getInteractionItemProps = (data: SeriesItemIdentifier) => {
-    const onMouseEnter = () => {
+    const onPointerDown = (event: React.PointerEvent) => {
+      if (event.currentTarget.hasPointerCapture(event.pointerId)) {
+        event.currentTarget.releasePointerCapture(event.pointerId);
+      }
+    };
```

Comment:
Could you explain what this code is doing?

I'm not familiar with those pointer specific events

Reply:
The `if` is mostly optional I believe, is just there for an early return if the pointer has been released. But in general,

```
if :element is capturing all the pointer events for pointer with :id
then release pointer capture for :id so it can be captured by another element :element'
```

In effect what it does is allow a single touch to be captured by any element it passes over rather than just the first one. 😅